Package.swift

```diff
@@ -27,3 +27,9 @@ let package = Package(
             dependencies: ["CodeEditKit"]),
     ]
 )
+
+#if swift(>=5.6)
+package.dependencies += [
+    .package(url: "https://github.com/apple/swift-docc-plugin", from: "1.0.0"),
+]
+#endif
```

Comment:
I would rather change the Swift version for `CodeEditKit` to `5.6`

Reply:
Got it.